### PR TITLE
0.8.6 Testing 1

### PR DESCRIPTION
**Pull request by wanda-carlson**
0.8.6 Testing 1

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Here's an image. 
Screen Shot 2022-01-06 at 7 46 07 AM

**Comment by wanda-carlson** [comment]
Isn't it?

**Comment by wanda-carlson** [comment]
**Slack user** hello

**Comment by wanda-carlson** [comment]
**Slack user** hi

**Comment by wanda-carlson** [comment]
**Slack user** hello

**Comment by wanda-carlson** [comment]
This is nice @wanda-carlson

**Comment by henry-richard** [comment]
HI @wanda-carlson

**Comment by henry-richard** [comment]
Hi @wanda-carlson

**Comment by henry-richard** [comment]
@wanda-carlson hello

**Comment by henry-richard** [comment]
Hi @wanda-carlson

**Comment by henry-richard** [comment]
@wanda-carlson hello there

File: README.md [hunk 1]
```diff
@@ -40,3 +40,4 @@ print("WANDA 2")
 print("WANDA 2") 
 print("WANDA 3") 
 print("WANDA 4")
```
Comment (henry-richard) [inline README.md:42]: this file is getting big

File: README.md [hunk 2]
```diff
@@ -40,3 +40,4 @@ print("WANDA 2")
 print("WANDA 2") 
 print("WANDA 3") 
 print("WANDA 4") 
+0.8.6 Testing 1
```
Comment (henry-richard) [inline README.md:43]: this looks good

**Review by henry-richard — CHANGES_REQUESTED** [review]
@wanda-carlson minor changes

**Review by henry-richard — APPROVED** [review]
This is better

**Pull request merged**